src/modules/playground.ts

```diff
@@ -200,7 +200,7 @@ function getSelectionQueryParams(query: string, numLines: number) {
 	const [startLine, endLine] = ['pln', 'ssl']
 		// @ts-expect-error parseInt(null) is okay here since we check for NaN
 		.map(name => parseInt(params.get(name)))
-		.map(n => (n !== NaN && 1 <= n && n <= numLines ? n : undefined))
+		.map(n => (!Number.isNaN(n) && 1 <= n && n <= numLines ? n : undefined))
```

Comment:
Just a quick fix, since the code before wasn't buggy either (`1 <= NaN` is `false`). Because `NaN !== NaN`, the code before actually didn't do anything.